### ISSUE TITLE
Version 2.13.1 suddenly requires newer glibc version

### ISSUE THREAD

**Issue by tmaarse** [meteor/meteor#12771]
The following `Dockerfile` fails to build with Meteor 2.13.1, while it does work when changing the version to 2.13.0:

```dockerfile
FROM node:14-slim

RUN npm install -g meteor@2.13.1 --unsafe-perm

ENV PATH="/root/.meteor:$PATH"
ENV METEOR_ALLOW_SUPERUSER=true

RUN meteor create simple-todos-react --prototype
```

Using `meteor/node:14.21.4` as base image doesn't help either. The following error is generated in both cases:

```
$ docker build .
# ...
Step 4/4 : RUN meteor create simple-todos-react --prototype
 ---> Running in 24666f81016a
internal/modules/cjs/loader.js:1175
  return process.dlopen(module, path.toNamespacedPath(filename));
                 ^

Error: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /root/.meteor/packages/meteor-tool/.2.13.1.gadqg5.bafha++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/vscode-nsfw/build/Release/nsfw.node)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:1175:18)
    at Module.load (internal/modules/cjs/loader.js:981:32)
    at Module.Mp.load (/tools/static-assets/server/runtime.js:35:31)
    at Function.Module._load (internal/modules/cjs/loader.js:821:12)
    at Module.require (internal/modules/cjs/loader.js:1005:19)
    at require (internal/modules/cjs/helpers.js:107:18)
    at Object.<anonymous> (/root/.meteor/packages/meteor-tool/.2.13.1.gadqg5.bafha++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/vscode-nsfw/js/src/index.js:4:14)
    at Module._compile (internal/modules/cjs/loader.js:1116:14)
    at Module.Mp._compile (/tools/static-assets/server/runtime.js:78:21)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1145:10) {
  code: 'ERR_DLOPEN_FAILED'
}
The command '/bin/sh -c meteor create simple-todos-react --prototype' returned a non-zero code: 1
```

In our project `meteor build` fails as well, so it's not only `create` which is failing.

The changed `libc` requirement isn't listed in the changelog, so this seems a regression.

OS: Ubuntu 23.04 x86_64, 6.2.0-26-generic

**Comment by a4xrbj1** [comment]
Same happens from me but I'm still on Meteor version 2.12 so I don't think this is coming from 2.13.1. 

`
[Container] 2023/09/07 19:04:46 Running command meteor build build/ --allow-superuser --directory --architecture os.linux.x86_64
--
192 | internal/modules/cjs/loader.js:1175
193 | return process.dlopen(module, path.toNamespacedPath(filename));
194 | ^
195 |  
196 | Error: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /root/.meteor/packages/meteor-tool/.2.13.1.1y6omdi.rbzs++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/vscode-nsfw/build/Release/nsfw.node)
197 | at Object.Module._extensions..node (internal/modules/cjs/loader.js:1175:18)
198 | at Module.load (internal/modules/cjs/loader.js:981:32)
199 | at Module.Mp.load (/tools/static-assets/server/runtime.js:35:31)
200 | at Function.Module._load (internal/modules/cjs/loader.js:821:12)
201 | at Module.require (internal/modules/cjs/loader.js:1005:19)
202 | at require (internal/modules/cjs/helpers.js:107:18)
203 | at Object.<anonymous> (/root/.meteor/packages/meteor-tool/.2.13.1.1y6omdi.rbzs++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/vscode-nsfw/js/src/index.js:4:14)
204 | at Module._compile (internal/modules/cjs/loader.js:1116:14)
205 | at Module.Mp._compile (/tools/static-assets/server/runtime.js:78:21)
206 | at Object.Module._extensions..js (internal/modules/cjs/loader.js:1145:10) {
207 | code: 'ERR_DLOPEN_FAILED'
208 | }
209 |  
210 | [Container] 2023/09/07 19:04:47 Command did not exit successfully meteor build build/ --allow-superuser --directory --architecture os.linux.x86_64 exit status 1
211 | [Container] 2023/09/07 19:04:47 Phase complete: BUILD State: FAILED
212 | [Container] 2023/09/07 19:04:47 Phase context status code: COMMAND_EXECUTION_ERROR Message: Error while executing command: meteor build build/ --allow-superuser --directory --architecture os.linux.x86_64. Reason: exit status 1
`

**Comment by conorstrejcek** [comment]
@a4xrbj1 It is coming from the `2.13.1` installation of meteor, you can see it in your error:

```
required by /root/.meteor/packages/meteor-tool/.2.13.1.1[...]
```

This is distinct from the version of meteor your application runs on.

It seems like the fix made for https://github.com/meteor/meteor/issues/12731 has a regression on Linux boxes.

**Comment by tmaarse** [comment]
Confirmed fix in 2.13.3
